environments/pull-environments.sh

```diff
@@ -16,22 +84,21 @@ for dir in ./*/ ; do
 done
 
 # max number of parallel tasks at a time
-numParallelTasks=8; # Future improvement: could set this to # of logical cores in localdevice
+numParallelTasks=8 # Future improvement: could set this to # of logical cores in localdevice
```

Comment:
I saw you change the number back in f7e32dccc933bb169fe42635a850e603eb76963e, curious what happened

Reply:
I was falling asleep and messed up the number while I was working locally 😅